Guide To Contribute To Open Source.md

```diff
@@ -0,0 +1,66 @@
+# Beginners Guide To Contribute To Open Source
+What is ************Open Source************: Open source is anything that is created by person which could be a "Software" Or "Code" that is publicly accessible to users like ****you****. Doing this will allow you to "Modify" "Alter" "Change" The way you see fit. 
+
+What is the point of ************Open Source************: This allows users to take peices of that "Software" or "Code" and change it for their own liking, which then if the main creators believes that the "Software" Or "Code" is a greater than the oringal creation then it can be integrate into a bigger project or changed from the orginal creation.    
+
+
+
+
+## Steps on how to contribute to a open source Repositories 
+
+1. When wanting to contribute to open source you will need to fork a repository. A fork is how you copy and paste what ever files that are accessible to you. To fork you will need to goto the bit project and click the fork on the top right of the page. Once you do that it will copy the files to your repository
+
+![Image To Show Fork](https://docs.github.com/assets/images/help/repository/fork_button.jpg)
+
+2. You will then need to click on your profile link on the top right of the page and click my Repository once there you’ll see a copy version of bit project. 
```

Comment:
you need to italicize or add quotations to things you need to click, in this case my repository. 

copy version of bit project? you mean the repo?